data/migrations/20190212173915_notes_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('notes_table',
+  notes => {
+    
+    notes.increments();
+    notes.string('note_title').notNullable();
+    notes.string('note_content');
+
+  })
+};
+
+exports.down = function(knex, Promise) {
+    return knex.schema.dropTableIfExists('notes_table');
+};
```

Comment:
I personally say delete old code (even migrations) if all it serves is being a history; you already have that via Git.